src/ffpuppet/minidump_parser.py

```diff
@@ -121,37 +134,63 @@ def format_output(
         if limit < len(frames):
             out_fp.write(b"WARNING: Hit stack size output limit!\n")
 
-    def to_json(self, src: Path, dst: Path) -> Path:
+    def to_json(self, src: Path) -> Tuple[bool, Path]:
         """Convert a minidump to json a file.
 
         Args:
             src: Minidump file.
-            dst: Location to save JSON file.
 
         Returns:
-            A file containing JSON output.
+            Status and a file containing minidump-stackwalk output.
         """
         assert self.MDSW_BIN
         cmd = [self.MDSW_BIN, "--no-color", "--json"]
-        if self._symbols_path:
-            cmd.extend(["--symbols-path", str(self._symbols_path)])
+        if self._symbols:
+            cmd.extend(["--symbols-path", str(self._symbols)])
         else:
             cmd.extend(["--symbols-url", "https://symbols.mozilla.org/"])
-        # add output files
-        with NamedTemporaryFile(dir=dst, prefix="mdsw_out_", suffix=".json") as ofp:
-            out_json = Path(ofp.name)
-        cmd.extend(["--output-file", str(out_json)])
         cmd.append(str(src))
+        # get unique file name
+        with NamedTemporaryFile(dir=self._storage, prefix="mdsw_out_") as ofp:
+            out_json = Path(ofp.name)
         LOG.debug("running %r", " ".join(cmd))
         try:
-            run(cmd, check=True, stdout=DEVNULL, timeout=60)
-        except CalledProcessError:
+            with out_json.open("wb") as out_fp:
+                run(cmd, check=True, stderr=out_fp, stdout=out_fp, timeout=60)
```

Comment:
Redirecting stderr means any warning message is going to corrupt the json output?